Makefile

```diff
@@ -63,6 +63,23 @@ image-proxy: proxy
 	docker pull registry.access.redhat.com/ubi8/ubi-minimal
 	docker build -f Dockerfile.proxy -t arosvc.azurecr.io/proxy:latest .
 
+image-aro-int: aro
+	docker pull registry.access.redhat.com/ubi8/ubi-minimal
+	docker build -f Dockerfile.aro -t arointsvc.azurecr.io/aro:$(COMMIT) .
+
+image-mdm-int:
+	docker build --build-arg VERSION=2.2019.801.1228-66cac1-~bionic_amd64 \
+	  -f Dockerfile.mdm -t arointsvc.azurecr.io/mdm:2019.801.1228-66cac1 .
+
+image-proxy-int: proxy
```

Comment:
Is our int env should be production confguration? If so, do we need proxy there? as Proxy is used for dev flow mostly.

Reply:
It uses production config so proxy isn't needed. But I would assume dev environment will move over to using the int ACR long term